test/services/consent/services/consentCheck.test.js

```diff
@@ -0,0 +1,147 @@
+const assert = require('assert');
+const { expect } = require('chai');
+const app = require('../../../../src/app');
+const testObjects = require('../../helpers/testObjects')(app);
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
Codacy found an issue: [`@schul-cloud/commons` import should occur before import of `../../../../src/app`](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5680233)